apps/sms/views/conversation/test/unit/link_helper_test.js

```diff
@@ -153,12 +153,14 @@ suite('link_helper_test.js', function() {
       test('non ascii prefix', function() {
         testURLMatch('àmozilla.org', 'mozilla.org', true);
       });
+      test('Bug 941763: Add dot as URL prefix', function() {
```

Comment:
Then in that way code will full of bug no. Also new contributor doesn't look much on old bug number. Mentors help them to get required info. Also it's not consistent. Also, IMO for new contributor, more important is..what piece of code is doing instead of how and from where it came from.
We do write bug no. as comments in case of big/critical patch or patch where more works need to be done in other Bug. Writing Bug no. In code looks bit weird. Commit details is already there is to show bug no.